src/System.Runtime.Extensions/ref/System.Runtime.Extensions.csproj

```diff
@@ -14,5 +14,9 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <!-- ToDo: Remove this dependency and use System.Runtime package once published -->
```

Comment:
Meant to do this once merged -- sorry. https://github.com/dotnet/corefx/issues/11093